specs/2025-02-24.yaml

```diff
@@ -1023,137 +938,411 @@ paths:
+          schema:
+            type: string
+      responses:
+        "200":
+          $ref: "#/components/responses/ListDisbursementsResponse"
+        "400":
+          $ref: "#/components/responses/BadRequestError"
+        "401":
+          $ref: "#/components/responses/AuthenticationError"
+        "403":
+          $ref: "#/components/responses/ForbiddenError"
+        "409":
+          $ref: "#/components/responses/ConflictError"
+        "500":
+          $ref: "#/components/responses/InternalServerError"
+  /v1/disbursements/{id}:
+    get:
+      summary: Retrieve a disbursement
+      description: |
+        Get a disbursement by its unique identifier.
+      operationId: getDisbursement
+      tags:
+        - disbursements
+      security:
+        - bearerAuth: []
+      parameters:
+        - name: id
+          in: path
+          description: The unique identifier for the disbursement
+          required: true
+          schema:
+            type: string
+      responses:
+        "200":
+          description: The disbursement was retrieved
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Disbursement"
+        "400":
+          $ref: "#/components/responses/BadRequestError"
+        "401":
+          $ref: "#/components/responses/AuthenticationError"
+        "403":
+          $ref: "#/components/responses/ForbiddenError"
+        "404":
+          $ref: "#/components/responses/NotFoundError"
+        "409":
+          $ref: "#/components/responses/ConflictError"
+        "500":
+          $ref: "#/components/responses/InternalServerError"
 components:
   securitySchemes:
     bearerAuth:
       type: http
       scheme: bearer
   schemas:
-    Nonprofit:
+    OrganizationSummary:
       type: object
       description: |-
-        A Nonprofit represents a registered 501(c)(3) charitable organization in good standing with the IRS.
+        A summary of an organization.
       required:
         - id
-        - name
         - ein
+        - name
       properties:
         id:
           type: string
-          readOnly: true
-          description: The unique identifier for the object.
-          example: g33ee4ee-f881-42a0-85e3-1dde7118w9k2
+          description: The unique identifier for the organization.
+          example: org_01j8rs605a4gctmbm58d87mvsj
+        ein:
+          type: string
+          description: The Employer Identification Number (EIN) or TaxID for the nonprofit entity.
+          example: 123456789
         name:
           type: string
-          description: The IRS registered name of the nonprofit organization
-          example: Leading for Children
-        preferredName:
+          description: The name of the organization.
+          example: American Red Cross
+        daf_eligible:
+          type: boolean
+          description: Whether the organization is eligible to receive donations from a Donor Advised Fund.
```

Comment:
Right now in payments the /search endpoint returns `is_daf_eligible`

It also includes
```
is_pub78_verified
is_ofac_flagged
is_cftb_flagged
is_splc_flagged
is_cagr_flagged
is_eligible_foundation_code
```

do we want to include that in the docs? remove it from the /search response object?

Reply:
i don't think we want to include the rest of these in the external facing search endpoint. we might want to have an internal version that includes these... will need to sync with Salo